core/invoker/src/main/scala/whisk/core/container/WhiskContainer.scala

```diff
@@ -92,17 +96,53 @@ class WhiskContainer(
     }
 
     /**
+     * Starts the tracing.
+     */
+    private def startSpan(msg: ActivationMessage): Option[Span] = {
+        val tracerOption = if (GlobalTracer.isRegistered) Option.empty else Some(TracerResolver.resolveTracer)
+        tracerOption foreach GlobalTracer.register
```

Comment:
It might be that `tracerOption` is null at this point:

https://travis-ci.org/apache/incubator-openwhisk/builds/245871954#L6909-L6913
```
    java.lang.NullPointerException: Cannot register GlobalTracer <null>.
        at io.opentracing.util.GlobalTracer.register(GlobalTracer.java:83)
        at whisk.core.container.WhiskContainer$$anonfun$startSpan$1.apply(WhiskContainer.scala:104)
        at whisk.core.container.WhiskContainer$$anonfun$startSpan$1.apply(WhiskContainer.scala:104)
        at scala.Option.foreach(Option.scala:257)
```